Server/docker-compose.yaml

```diff
@@ -28,6 +28,17 @@ services:
       - 4004
     command: settings-tp -C tcp://validator:4004
     stop_signal: SIGKILL
+    logging:
+        driver: loki
+        options:
+            loki-url: https://153011:eyJrIjoiNTBjYTkzYzk0NmQ3NDdlMjA5ZmMyMTZlMzRhYzZkN2Y1NjBlZGI1MSIsIm4iOiJjY29pbmRldiIsImlkIjo1OTEzMDB9@logs-prod3.grafana.net/loki/api/v1/push
+            loki-external-labels: container_name=sawtooth-settings-tp-testnet
+            loki-relabel-config: |
+              - action: labelmap
+                regex: compose_service
+                replacement: service
+            max-file: "5"
+            max-size: "10m"
```

Comment:
For now I don't think we're interested in collecting metrics from settings-tp but I guess it doesn't hurt.

The containers which are most important for now are validator, consensus and processor.